spec-0007/index.md

```diff
@@ -5,69 +5,60 @@ author:
   - "Stéfan van der Walt <stefanv@berkeley.edu>"
   - "Sebastian Berg <sebastianb@nvidia.com>"
   - "Pamphile Roy <roy.pamphile@gmail.com>"
+  - "Matt Haberland <mhaberla@calpoly.edu>"
   - Other participants in the discussion <not.yet@named.org>"
 discussion: https://github.com/scipy/scipy/issues/14322
 endorsed-by:
 ---
 
 ## Description
 
-<!--
-Briefly and clearly describe the proposal.
-Explain the general need and the advantages of this specific proposal.
-If relevant, include examples of how the new functionality would be used,
-intended use-cases, and pseudo-code illustrating its use.
--->
-
-There is disparity in the APIs libraries provide to seed random number generation.
-This SPEC suggests a single, pragmatic API for the ecosystem, taking into account technical and historical factors.
+Currently, libraries across the ecosystem provide various APIs for seeding random number generation.
+This SPEC suggests a unified, pragmatic API, taking into account technical and historical factors.
 Adopting such a uniform API will simplify the user experience, especially for those who rely on multiple projects.
 
-We recommend to:
+We recommend:
```

Comment:
This is not necessarily better, but I ran into a merge conflict, so I resolved it with what I had.